proto/api_spec/tracking.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+
+package api_spec;
+
+option go_package = "akitasoftware.com/superstar/pb/go/api_spec";
+
+import "google/protobuf/timestamp.proto";
+
+message AkitaWitnessTracking {
+  // Earliest recorded occurrence, inclusive.
+  google.protobuf.Timestamp first_seen = 1;
+
+  // Most recent recorded occurrence, inclusive, as an offset in seconds
+  // from first_seen.
+  uint32 last_seen_offset_seconds = 2;
```

Comment:
I think the reason you picked an offset rather than another Timestamp is a size optimization; the zero offset (which is all that we will see in most witnesses) is encoded by omission.  Should probably document that if that's what is intended.

Can we do the same for count = 1?

Is there a way to reduce the cost of the Timestamp itself?  (I see it uses nanosecond precision, but I wouldn't feel good about using a 32-bit time this close to 2038...)

Reply:
Yes, that's right.  I'll add a note to the comment.  And that's a good idea for `count = 1`, I'll make that change.

For `first_seen`, we could use a timestamp with our own epoch, since we know we'll never see a `first_seen` date prior to when we deploy this feature -- it could be something like "seconds since Feb 1, 2022".  Using 32 bits gives us coverage for the next 136 years.

Reply:
I guess we're not supporting this for uploaded traces, then.